app/ts/components/simulationExplaining/customExplainers/GovernanceVoteVisualizer.tsx

```diff
@@ -38,7 +42,57 @@ function MissingAbi(params: MissingAbiParams) {
 	</div>
 }
 
-export function GovernanceVoteVisualizer(param: TransactionImportanceBlockParams) {
+
+export function VotePanel({ inputParams }: { inputParams: GovernanceVoteInputParameters }) {
+	const interpretSupport = (support: bigint | boolean) => {
+		if (support === true || support === 1n) return 'For'
+		if (support === false || support === 0n) return 'Against'
+		if (support === 2n) return 'Abstain'
+		return `Support: ${ support }`
+	}
+
+	return <>
+		<div class = 'notification transaction-importance-box'>
+			<div style = 'display: flex; justify-content: center;' >
+				<p style = { { 'font-size': BIG_FONT_SIZE } } >
+					Vote&nbsp;<b>{ interpretSupport(inputParams.support) }</b>&nbsp;{`for proposal: ${ inputParams.proposalId } `}
+				</p>
+			</div>
+		</div>
+		
+		{ inputParams.reason !== undefined || inputParams.signature !== undefined || inputParams.voter !== undefined || inputParams.params !== undefined ? <>
+			<div class = 'container'>
+				<span class = 'log-table' style = 'justify-content: center; column-gap: 5px; row-gap: 5px; grid-template-columns: auto auto'>
+					{ inputParams.reason !== undefined ? <> 
+						<CellElement text = 'Reason:'/>
+						<CellElement text =  { inputParams.reason }/>
+					</> : <></> }
+					{ inputParams.signature !== undefined ? <> 
+						<CellElement text = 'Signature:'/>
+						<CellElement text = { dataStringWith0xStart(inputParams.signature) } />
+					</> : <></> }
+					{ inputParams.voter !== undefined ? <> 
+						<CellElement text = 'Voter:'/>
+						<CellElement text = { checksummedAddress(inputParams.voter) } /> 
+					</> : <></> }
+					{ inputParams.params !== undefined ? <>
+						<CellElement text = 'Params: '/>
```

Comment:
"Params" isn't particularly meaningful to regular humans.  I think "Additional Info" will be a bit more clear to them.
```suggestion
						<CellElement text = 'Additional Info: '/>
```
Alternatively:
```suggestion
						<CellElement text = 'Additional Data Included With Your Vote: '/>
```
My hope is that if a user sees some data in here they will stop and think about it.  I don't know how governance systems use this extra data, but it is worrying to me that it could be exploited somehow so I would rather users be cautious of `Params` on votes.

Reply:
I haven't seen it used for things I have been testing with